### PR TITLE
fix last of placeholders in #3741

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b8646d1](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/b8646d18a3ce3bd85270617081e37ff2c21c9d1b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b8646d1](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/b8646d18a3ce3bd85270617081e37ff2c21c9d1b)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 50 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b8646d1](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/b8646d18a3ce3bd85270617081e37ff2c21c9d1b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/4.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-5.0&branch=pr-en-us-3791) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.1/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-5.1&branch=pr-en-us-3791) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/6/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-6&branch=pr-en-us-3791) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903012040501539-3791%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=R9L1Xq5PsOUzCWPH1%2BTpPANaDVYyRkJIjddo789pdNM%3D&st=2019-03-01T20%3A50%3A08Z&se=2019-04-01T20%3A55%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[ec708ba](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/ec708baf344126f4000e1d3092ec86b9cca16c2c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[ec708ba](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/ec708baf344126f4000e1d3092ec86b9cca16c2c)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 50 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[ec708ba](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/ec708baf344126f4000e1d3092ec86b9cca16c2c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/4.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-5.0&branch=pr-en-us-3791) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.1/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-5.1&branch=pr-en-us-3791) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/6/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-6&branch=pr-en-us-3791) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903012041496545-3791%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mvdwifGSN5zy%2BB1S9%2Bq44Q7FSsRp7Om6OyjKs%2BRsEfE%3D&st=2019-03-01T21%3A04%3A46Z&se=2019-04-01T21%3A09%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[19b230c](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/19b230c7dd008dc9b8a49f3e09472053d17b1993)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[19b230c](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/19b230c7dd008dc9b8a49f3e09472053d17b1993)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 50 sec(s)_

File: reference/3.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md [hunk 1]
```diff
@@ -9,335 +9,341 @@ Module Guid:  eefcb906-b326-4e99-9f54-8b4bb6ef3c6d
...
 
 ### [New-WebServiceProxy](New-WebServiceProxy.md)
-Creates a Web service proxy object that lets you use and manage the Web service in Windows PowerShell.
 
+Creates a Web service proxy object that lets you use and manage the Web service in Windows PowerShell.
 
 ### [Pop-Location](Pop-Location.md)
-Changes the current location to the location most recently pushed onto the stack.
 
+Changes the current location to the location most recently pushed onto the stack. You can pop the location from the default stack or from a stack that you create by using the Push-Location cmdlet.
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md]: cmdlet should be code-fenced.

Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md]: fixed

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[19b230c](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/19b230c7dd008dc9b8a49f3e09472053d17b1993)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/4.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-5.0&branch=pr-en-us-3791) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-5.0&branch=pr-en-us-3791) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.1/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-5.1&branch=pr-en-us-3791) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.1/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-5.1&branch=pr-en-us-3791) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/6/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-6&branch=pr-en-us-3791) (powershell-6) |
[reference/6/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/6/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-6&branch=pr-en-us-3791) (powershell-6) |
[reference/6/Microsoft.WSMan.Management/Microsoft.WSMan.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/6/Microsoft.WSMan.Management/Microsoft.WSMan.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.wsman.management/index?view=powershell-6&branch=pr-en-us-3791) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903012059316547-3791%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=S3tdsj48K5Z43C9T00jwblBo9DJ309ee6IToGv9%2FOhs%3D&st=2019-03-01T21%3A19%3A15Z&se=2019-04-01T21%3A24%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[97b1312](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/97b1312fdc97bc4afd29faf9ad7a8aa1f55d87c3)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[97b1312](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/97b1312fdc97bc4afd29faf9ad7a8aa1f55d87c3)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 50 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[97b1312](https://github.com/sdwheeler/PowerShell-Docs/pull/3791/commits/97b1312fdc97bc4afd29faf9ad7a8aa1f55d87c3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/3.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-3.0&branch=pr-en-us-3791) (powershell-3.0) |
[reference/4.0/CimCmdlets/CimCmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/CimCmdlets/CimCmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/CimCmdlets/Register-CimIndicationEvent.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/CimCmdlets/Register-CimIndicationEvent.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/cimcmdlets/register-cimindicationevent?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/4.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-4.0&branch=pr-en-us-3791) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.0/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-5.0&branch=pr-en-us-3791) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.0/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-5.0&branch=pr-en-us-3791) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.1/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-5.1&branch=pr-en-us-3791) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/5.1/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-5.1&branch=pr-en-us-3791) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/6/Microsoft.PowerShell.Core/Microsoft.PowerShell.Core.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/index?view=powershell-6&branch=pr-en-us-3791) (powershell-6) |
[reference/6/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/6/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/index?view=powershell-6&branch=pr-en-us-3791) (powershell-6) |
[reference/6/Microsoft.WSMan.Management/Microsoft.WSMan.Management.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-cimfinal/reference/6/Microsoft.WSMan.Management/Microsoft.WSMan.Management.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.wsman.management/index?view=powershell-6&branch=pr-en-us-3791) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903012117376528-3791%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ZzMzafa0Hf05fdV9ZmGVYHHm6jIXxCNshU60bfR5G2E%3D&st=2019-03-01T21%3A33%3A35Z&se=2019-04-01T21%3A38%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**